src/blas/impl/KokkosBlas1_nrm1_spec.hpp

```diff
@@ -200,12 +200,23 @@ struct Nrm1<RV, XMV, 2, false, KOKKOSKERNELS_IMPL_COMPILE_LIBRARY> {
                                       : "KokkosBlas::nrm1[noETI]");
     const size_type numRows = X.extent(0);
     const size_type numCols = X.extent(1);
-    if (numRows < static_cast<size_type>(INT_MAX) &&
-        numRows * numCols < static_cast<size_type>(INT_MAX)) {
-      MV_Nrm1_Invoke<RV, XMV, int>(R, X);
+    if (numCols == size_type(1)) {
```

Comment:
If you don't mind I prefer `Kokkos::ArithTraits<size_type>::one()`

Reply:
👍 Done